docs/data/external-oasis-repos.md

```diff
@@ -63,94 +75,186 @@ root@host # chkconfig httpd on
+
+!!! note
+    This is an OSG-internal procedure
+
+1.  If there is a request from OSG Security to shut down the distribution of a repository, the OSG representative
+    needs to run `blank_osg_repository` on `oasis-replica` and give it the full name of the repository. This will
+    rename the repository directories to a name with the current timestamp and replace it with a blank repository.
+    It includes a step to run on the `oasis` machine, and attempts to do it with `ssh`, but if that fails it prints
+    instructions on how to finish by logging in to the `oasis` machine manually.
+2.  When it is time to put the repository back into production, the GOC representative runs `unblank_osg_repository`
+    on `oasis-replica` and gives it the full name of the repository again. This will find the directory with the old
+    timestamp and put it back into service. This step also attempts to `ssh` to the `oasis` machine.
+
+Policies
+--------
 
-Procedure to shut down and remove an external repository
---------------------------------------------------------
+To help us provide the best operational setup possible, we have a few replication policies worth mentioning here:
 
-1. First, if the repository has been replicated outside of the U.S., the repository service administrator should open a GGUS ticket asking that the replication be removed from EGI stratum 1s. Wait until they say they are finished before going to the next step. 
-1. Next, the repository service administrator opens a GOC ticket asking to shut down the repository, giving the repository name, for example `repo.domain.name`, and the VO it belongs to. 
-1. After validating that the ticket submitter is authorized by a registered OASIS manager, the GOC representative next deletes the registered value for `repo.domain.name` in OIM for the VO in OASIS Repo URLs. 
-1. Next, the GOC representative adds the FNAL and BNL stratum 1 administrators to the ticket to asks them to remove the repository. 
-1. When the FNAL and BNL stratum 1 administrators say they have finished, the GOC representative then runs `cvmfs_server rmfs -f repo.domain.name` and `rm -r /oasissrv/cvmfs/repo.domain.name` on oasis-replica-itb and oasis-replica. 
-1. Finally, the GOC representative does `rm -r /srv/cvmfs/repo.domain.name` on oasis-itb and oasis.
+1.  OSG Operations only hosts the shared `oasis.opensciencegrid.org` repository; VO-dedicated software respositories
+    (such as `nova.opensciencegrid.org` for the NoVA VO) should be operated by the VO.
+2.  VOs are asked to either run their own repository or utilize the shared repository, but not both.
+3.  There is a finite amount of high-performance storage on the CDN.   A minimum of 100 GB per repository is guaranteed.
+    Larger limits may be requested.
+4.  VOs may ask the OSG to replicate their repositories to the European Grid Infrastructure (EGI); however, this can
+    only be done if the repository name ends in `.opensciencegrid.org`.
```

Comment:
Should we throw this section into https://opensciencegrid.github.io/technology/policy/external-oasis-repos/?

Reply:
Done: https://github.com/opensciencegrid/technology/pull/362